internal/kafka/internal/clusters/ocm_provider.go

```diff
@@ -456,12 +466,44 @@ func (o *OCMProvider) GetQuotaCosts() ([]types.QuotaCost, error) {
 		return quotaCostList, errors.New("failed to get quota cost: organisation id for the current authenticated user can't be found")
 	}
 
-	ocmQuotaCostList, err := o.ocmClient.GetQuotaCosts(orgID, false, false)
+	strimziOperatorAddonName := fmt.Sprintf("%s-%s", AMSQuotaAddonResourceNamePrefix, o.ocmConfig.StrimziOperatorAddonID)
+	fleetshardOperatorAddonName := fmt.Sprintf("%s-%s", AMSQuotaAddonResourceNamePrefix, o.ocmConfig.KasFleetshardAddonID)
+
+	filters := []ocm.QuotaCostRelatedResourceFilter{
+		// Filter for Strimzi operator add-on
+		{
+			ResourceName: &strimziOperatorAddonName,
+		},
+		// Filter for Fleetshard operator add-on
+		{
+			ResourceName: &fleetshardOperatorAddonName,
+		},
+		// Filter for osd compute nodes
+		{
+			ResourceType: &AMSQuotaComputeNodeResourceType,
+			Product:      &AMSQuotaOSDProductName,
+		},
+		// Filter for osd cluster
+		{
+			ResourceType: &AMSQuotaClusterResourceType,
+			Product:      &AMSQuotaOSDProductName,
+		},
+	}
+
+	// Only include cluster logging operator filter if its specified
+	// If not specified, this won't install on the data plane cluster so we don't need to retrieve the quota cost information
+	if o.ocmConfig.ClusterLoggingOperatorAddonID != "" {
```

Comment:
As a side comment there is current work to move cluster logging addon installation to be a SyncSet instead of an addon so maybe not worth tracking this?

Reply:
Thanks for pointing that out. +1 for me, i don't think it's worth tracking this. I'll remove it.